### DISCUSSION TITLE
Best way to load a complete show

### DISCUSSION THREAD

**Discussion by schaechtele** [trakt/trakt-api#255] [Q&A] [answered]
Hi,

I just wanted to ask if there's a best practice on how to load a complete show with all season and episode information to have the least api calls triggered. On an initial sync in my app, I'm gathering all the information only once (and do incremental updates after that), but for users with many watched shows, the API limit is reached when doing the initial sync, especially for loading the episodes (which I have to do because of the episode first aired date).

So in general I'm currently doing the following:
1. `shows/id` to get show information
2. `shows/id/seasons` **without** `?extended=episodes` set, to get all seasons
3. `shows/id/seasons/season` **with** `?extended=full` set, to get the episodes for each season including `first_aired`

If I wouldn't need `first_aired` for episodes, I could call `shows/id/seasons` with `?extended=episodes` set. Do you see any other way to accomplish this?

Thanks!

Sebastian

**Comment by rectifyer** [comment, marked as answer]
You can send multiple parameter to the `/shows/id/seasons` like `?extended=episodes,full`

**Reply by schaechtele** [reply]
Thanks! This is a very helpful piece of information.
